docs/Troubleshooting.md

```diff
@@ -53,14 +53,15 @@ function todos(state = [], action) {
       ]
     case 'COMPLETE_TODO':
       // Return a new array
-      return [
-        ...state.slice(0, action.index),
-        // Copy the object before mutating
-        Object.assign({}, state[action.index], {
-          completed: true
-        }),
-        ...state.slice(action.index + 1)
-      ]
+      return state.map((todo, index) => {
+        if(index === action.index) {
```

Comment:
Nit: we leave space after `if` pretty much everywhere else.
Would you mind fixing this as a separate PR?